deltacat/tests/compute/compact_partition_test_cases.py

```diff
@@ -1087,9 +1092,31 @@ def with_compactor_version_func_test_param(
     ),
 }
 
-INCREMENTAL_TEST_CASES = with_compactor_version_func_test_param(INCREMENTAL_TEST_CASES)
 
+NO_RCF_OUTPUT_TEST_CASES: Dict[str, NoRCFOutputCompactionTestCaseParams] = {
```

Comment:
We have to ideally add a test case for rebase then incremental where incremental contains delta without any files in it.